backend/src/v2/compiler/argocompiler/dag.go

```diff
@@ -570,10 +571,14 @@ func (c *workflowCompiler) addDAGDriverTemplate() string {
 		"--http_proxy", proxy.GetConfig().GetHttpProxy(),
 		"--https_proxy", proxy.GetConfig().GetHttpsProxy(),
 		"--no_proxy", proxy.GetConfig().GetNoProxy(),
+		"--ml_pipeline_service_tls_enabled", strconv.FormatBool(c.mlPipelineServiceTLSEnabled),
```

Comment:
I don't think this one is needed due to the `if c.mlPipelineServiceTLSEnabled {` block below.
```suggestion
```